### PR TITLE
Fix Read-me and Consecutive-Command

### LINKED ISSUE #156: [Consecutive Command] Bug occurs when wrong command added first
When a command is entered but the input is wrong, it doesnt allow you to correct that input and retry

Add <Wrong input>

"Invalid input"

Add <Correct input>
"Command: Add has been executed already"

### PR DESCRIPTION
**Pull request by martout2002**
fixes #156

### CODE REVIEW

**Review by Hoodini231 — CHANGES_REQUESTED** [review]
LGTM just need fix readme prefixs

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,20 +72,24 @@ GoonBook Level 3 (AB3) is a **desktop app for managing students, optimized for u
...
+![help message](images/Help.png)
+
+
+
 ### Adding a person: `add`
 
 Adds a person to the address book.
 
-Format: `add n/NAME c/CLASS p/PHONE_NUMBER [t/TAG]…​`
+Format: `add /name NAME /class CLASS /phone PHONE_NUMBER [t/TAG]…​`
```
Comment (wenjebs) [inline docs/UserGuide.md:85]: remember to change this prefix as well!

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -63,69 +64,87 @@ public Command parseCommand(String userInput) throws ParseException {
...
-            return new ConsecutiveCommand(commandWord);
+        if (commandWord.equals(lastCommandWord) & arguments.equals(lastArgument)) {
+            return new ConsecutiveCommand(commandWord, arguments);
         }
-
-        // Update lastCommandWord to the current command
-        lastCommandWord = commandWord;
-
-
+        Command command;
```
Comment (wenjebs) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:70]: lgtm

**Review by Hoodini231 — APPROVED** [review]

**Comment by Hoodini231** [comment]
Thank you for the revisions Martin !

**Pull request merged**